src/main/java/org/apache/sysml/runtime/instructions/SPInstructionParser.java

```diff
@@ -63,6 +63,7 @@
 import org.apache.sysml.runtime.instructions.spark.MultiReturnParameterizedBuiltinSPInstruction;
 import org.apache.sysml.runtime.instructions.spark.PMapmmSPInstruction;
 import org.apache.sysml.runtime.instructions.spark.ParameterizedBuiltinSPInstruction;
+import org.apache.sysml.runtime.instructions.spark.ParamservBuiltinSPInstruction;
```

Comment:
As discussed before, I would recommend to delete this unnecessary spark instruction. Looking forward, the central CP paramserv function will also allow us later to automatically determine the mode in an automatic manner on execute.